### DISCUSSION TITLE
After wordpress database update I see this error on my site

### DISCUSSION THREAD

**Discussion by district6wc** [code4recovery/12-step-meeting-list#1581] [General]
I had to update the wordpress database on my site and after that completed I now have this error on all pages

Notice: Function _load_textdomain_just_in_time was called incorrectly. Translation loading for the 12-step-meeting-list domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the init action or later. Please see [Debugging in WordPress](https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/) for more information. (This message was added in version 6.7.0.) in /var/www/wp-includes/functions.php on line 6114.

I have tried hard refreshes and all my plugins have been updated but the message persists. Is there a fix for this issue?

Thanks
Rose 
District 6
NC Bermuda

**Comment by joshreisner** [comment]
hi @district6wc - it sounds like your site is in debug mode for some reason. this should be disabled in production. go to `wp-config.php` and make sure WP_DEBUG is set to `false`:

```php
define( 'WP_DEBUG', false );
```

we'll be releasing a fix soon for this issue as well so you'll be able to have debug mode enabled and not see this message

**Comment by district6wc** [comment]
Thanks again
